olive/engine/engine.py

```diff
@@ -978,14 +978,13 @@ def _run_pass(
         run_end_time = datetime.now().timestamp()
         logger.info("Pass %s:%s finished in %f seconds", pass_id, pass_name, run_end_time - run_start_time)
 
-        if not enable_cloud_cache:
```

Comment:
We still need local cache for [no search run](https://github.com/microsoft/Olive/blob/main/olive/engine/engine.py#L472)